lms/static/js/edxnotes/views/tab_view.js

```diff
@@ -116,9 +116,7 @@ define([
         showErrorMessage: function (message) {
             this.$('.wrapper-msg')
                 .removeClass('is-hidden')
-                .find('.msg-content .copy').text(message)
-                .closest('.msg')
-                .focus();
```

Comment:
@polesye, @talbs what sort of error message is this?  If its just static text (nothing interactive), then you can just put an aria-live region on the div.  For this to work (best), the div should already be present in the DOM with the aria-live="polite" attribute, and then the error msg is inserted using JS.  Again, only for non-interactive messages.